modules/Bio/EnsEMBL/Production/Pipeline/Common/Base.pm

```diff
@@ -739,4 +739,15 @@ sub has_genes {
   return $count;
 }
 
+sub get_biotype_group {
+  my ($self, $group) = @_;
+
+  my $ba = Bio::EnsEMBL::Registry->get_adaptor($self->param('species'), 'core', 'Biotype');
+
+  my $biotypes = $ba->fetch_all_by_group_object_db_type( $group, 'gene', 'core' );
```

Comment:
this method is not available in the BiotypeAdaptor

Reply:
The implementation is pending merge of https://github.com/Ensembl/ensembl/pull/230

Reply:
Unless I am mistaken, this should be ready now.

Reply:
yep, merging now